src/clr-angular/forms-deprecated/_forms.clarity.scss

```diff
@@ -28,7 +28,8 @@
   &.required {
     &:after {
       content: '*';
-      font-size: 1.1em;
+      font-size: 0.58479532rem; // Approximately 14px.
+      line-height: 0.5rem;
```

Comment:
Why do we need a line height here and why is it 12px?

Reply:
The `:after` pseudo element has a font size on it that is adding the pixel height. In order to preserve the same font size I added a line-height to keep it from pushing up and adding the extra pixel.